include/grpc++/server_builder.h

```diff
@@ -62,6 +62,22 @@ class ServerBuilder {
  public:
   ServerBuilder();
 
+  struct SyncServerSettings {
```

Comment:
This is a bit scary. Recommend one of two approaches:

A constructor for SyncServerSettings that fills in defaults (so that users don't need to configure *all* the options)

OR

An enumeration of options: enum { NumCQs, MinPollers, MaxPollers, CQTimeoutMsec } SyncServerOption;
And a SetSyncOption(SyncServerOption option, int value); method instead of an overall set.